legacy/Dockerfile

```diff
@@ -59,27 +63,28 @@ ENV PATH=/opt/gcc-arm-none-eabi-${TOOLCHAIN_VERSION}/bin:$PATH \
 # Adding cargo binaries to PATH
 ENV PATH=${CARGO_HOME}/bin:${PATH}
 
-# Installing rustup to manage rust toolchainsand setting the default toolchain to the RUST_NIGHTLY_VERSION nightly channel.
+# Installing rustup to manage rust toolchains and setting the default toolchain to the RUST_STABLE_VERSION stable channel.
 RUN curl https://sh.rustup.rs -sSf | \
-    sh -s -- --default-toolchain $RUST_NIGHTLY_VERSION -y
+    sh -s -- --default-toolchain $RUST_STABLE_VERSION -y
 
-# Installing stable channel version RUST_STABLE_VERSION as well (for cargo-ledger installation).
-RUN rustup install $RUST_STABLE_VERSION
+# Installing nightly channel version RUST_NIGHTLY_VERSION
+RUN rustup install $RUST_NIGHTLY_VERSION
 
-# Adding ARMV6M target to the installed nightly toolchain
-RUN rustup target add thumbv6m-none-eabi --toolchain $RUST_NIGHTLY_VERSION
+# Adding ARMV6M target to the installed stable toolchain
+RUN rustup target add thumbv6m-none-eabi --toolchain $RUST_STABLE_VERSION
```

Comment:
Don't we need this for stable **and** nightly ?

Reply:
Ok let's add it for both channels